### PR TITLE
Renovate stucture

### PR DESCRIPTION
**Pull request by simonspa**
@petricm have a look, open for suggestions.

### CODE REVIEW

**Comment by petricm** [comment]
- [x] add `travis.yml` (update to GCC8 and LLVM6)
- [x] remove `USE_CXX11` and steer CXX standard via CMAKE_CXX_STANDARD
- [x] r-path settings in ilcsoft_default_rpath_settings.cmake are bad should more like [Always full RPATH](https://gitlab.kitware.com/cmake/community/wikis/doc/cmake/RPATH-handling)
- [x] remove `FindROOT.cmake`
- [x] CMakeList.txt in top level folder
- [x] the top level WIKI file/link to www.wiki.terascale.de/index.php/GeneralBrokenLines does not work
- [x] `-Wl,-no-undefined` not suitable for clang use `-Wl,-undefined,error`

File: .travis-ci.d/compile_and_test.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+cd /Package
+mkdir build
+cd build
+source ../.travis-ci.d/init_x86_64.sh
+source $ROOTSYS/bin/thisroot.sh
+cmake -GNinja -DROOT_DIR=$ROOTSYS -DEigen3_DIR=$Eigen3_DIR .. && \
```
Comment (petricm) [inline .travis-ci.d/compile_and_test.sh]: add `-DCMAKE_CXX_FLAGS="-fdiagnostics-color=always"` to get color to ninja

File: .travis-ci.d/init_x86_64.sh [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+export Eigen_HOME=${CLICREPO}/software/Eigen/3.3.4/${BUILD_FLAVOUR}/
+export Eigen3_DIR=${Eigen_HOME}/share/eigen3/cmake/
+export CMAKE_PREFIX_PATH="$Eigen3_DIR:$CMAKE_PREFIX_PATH"
+
+#--------------------------------------------------------------------------------
+#     Doxygen
+#--------------------------------------------------------------------------------
+
+export Doxygen_HOME=${SFTREPO}/lcg/releases/doxygen/1.8.11-68a7c/x86_64-centos7-gcc7-opt/bin/
```
Comment (petricm) [inline .travis-ci.d/init_x86_64.sh]: Your container is SL6, and you can use `${CLICREPO}/software/Doxygen/1.8.14/`

Comment (simonspa) [reply .travis-ci.d/init_x86_64.sh]: I was quicker, found already.

**Comment by simonspa** [comment]
I think I'm done with this. Have a look and check if you spot anything else, @petricm

**Comment by petricm** [comment]
Looks good

**Comment by petricm** [comment]
You could also add the travis badge to the readme

**Comment by simonspa** [comment]
:racehorse:

**Pull request merged**